README.md

```diff
@@ -15,42 +15,42 @@ The `.cctemplate` is a [Template Descriptor](https://cloud.google.com/code/docs/
     },
     "templates": [
         {
-            "path": "path/to/my/template",
-            "name": "My Template",
-            "description": "This template helps you create something amazing!"
+            "path": "path/to/my/sample",
+            "name": "My Sample",
+            "description": "This sample helps you create something amazing!"
         },
         ...
     ]
 }
 ```
 
-There is an example `.cctemplate` file in this repository that you can review, and you can learn more about the Template Descriptor schema [here](https://cloud.google.com/code/docs/vscode/set-up-template-repo#template_descriptor_schema).
+There is an example `.cctemplate` file in this repository that you can review, and you can learn more about the Sample Descriptor schema [here](https://cloud.google.com/code/docs/vscode/set-up-template-repo#template_descriptor_schema).
 
 ### hello-world
 
-A sample golang template. You can see an entry for this template in the `.cctemplate` file.
+A simple sample written in Go. You can see an entry for this sample in the `.cctemplate` file.
 
-### kubernetes-custom-template/nodejs
+### kubernetes-custom-sample/nodejs
 
-A sample Node.js template with a frontend and backend service. This template uses MongoDB as a database and runs on Kubernetes. You can see an entry for this template in the `.cctemplate` file.
+A Node.js sample with a frontend and backend service. This sample uses MongoDB as a database and runs on Kubernetes. You can see an entry for this sample in the `.cctemplate` file.
 
 ---
 
 ## Usage
 
-To see an example of how the Custom Templates feature of Cloud Code works, you can configure your IDE to recognize this repository and create applications based on the templates within. Follow the steps below to see Custom Templates in action!
+To see an example of how the Custom Sample feature of Cloud Code works, you can configure your IDE to recognize this repository and create applications based on the sample within. Follow the steps below to see Custom Sample in action!
```

Comment:
"Follow the steps below to see Custom Sample**s**?

Reply:
or "this feature in action"

Reply:
"this feature in action" is perfect 👍